src/System.Threading.Thread/src/System/Threading/Thread.cs

```diff
@@ -123,6 +137,19 @@ private void RequireCurrentThread()
             }
         }
 
+        private void SetCultureOnUnstartedThread(CultureInfo value, ref CultureInfo culture)
```

Comment:
>SetCultureOnUnstartedThread [](start = 21, length = 27)

why this method doesn't return the value instead of passing another ref parameter? I am not sure if there is any perf difference here.

Reply:
I had this code duplicated originally. Stephen suggested this pattern (use `ref` argument) to factor out the common code. I think it is a matter of personal preference where it is better to use `ref` or return the value here. I do not have an opinion.